src/include/kernel/abi.h

```diff
@@ -29,7 +29,7 @@
 
 /** \brief SOF ABI version major, minor and patch numbers */
 #define SOF_ABI_MAJOR 3
-#define SOF_ABI_MINOR 18
+#define SOF_ABI_MINOR 19
 #define SOF_ABI_PATCH 1
```

Comment:
rev patch down?

Reply:
Sorry,  I'm missing the issue. Can you explain?

Reply:
@kv2019i Explained, got it now! I'll change PATCH to zero in next version.

Reply:
missed this in push